assets/idp.example.edu.dist/idp/conf/services.xml

```diff
@@ -37,9 +37,7 @@
     <util:list id ="shibboleth.AttributeFilterResources">
         <value>%{idp.home}/conf/metadata-based-attribute-filter.xml</value>
         <value>%{idp.home}/conf/attribute-filter.xml</value>
-        {% if enable_edugain == "true" %}
```

Comment:
I was just wondering why the removal of edugain here?

Reply:
The eduGAIN attribute filter just covered the R&S entity category. This have been replaced with a policy-based-attribute filter the handles all of the REFEDS entity categories.